tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     vmImage: 'macOS-10.13'
   steps:
   
-    - template: templates/set-test-data-variables-step.yml
+    - template: templates/mac-set-variables-and-download.yml
     - template: templates/set-version-number-variables-step.yml
```

Comment:
Line 150: this is still using CUDA 9.1. Next version of ORT will be on CUDA 10.0 -- should update (either in this PR or another PR).

Reply:
Hi @jignparm,  Your suggestion is good, but I'd prefer to keep this PR small and defer the change to another pr.  So that if something went wrong after the checkin, it would be easier to debug and fix the issue. Thank you.